app/models/gobierto_people/person_statement.rb

```diff
@@ -25,6 +25,7 @@ class PersonStatement < ApplicationRecord
     belongs_to :site
 
     scope :sorted, -> { order(published_on: :desc, created_at: :desc) }
+    scope :sorted_by_person_position, -> { joins(:person).order('gp_people.position ASC, published_on DESC, gp_person_statements.created_at DESC') }
```

Comment:
Could you use `GobiertoPeople::Person.table_name` and `self.class.table_name` instead of the name of the tables?

Reply:
Done!